src/shared/components/BandPlot.tsx

```diff
@@ -208,11 +213,9 @@ const BandPlot: FC<Props> = ({
   return children(config)
 }
 
-const mstp = (state: AppState) => {
-  const {builderConfig} = getActiveQuery(state)
-  const {functions} = builderConfig
-  return {selectedFunctions: functions.map(f => f.name)}
-}
+const mstp = (state: AppState) => ({
+  activeQueryIndex: getActiveQueryIndex(state),
```

Comment:
why use the index and not keep with the active query?

Reply:
Because in Dashboard view, the active query only keeps a single aggregate function in the draft queries. This causes the upper and lower columns to disappear from the cells. The upper and lower columns are available in the props to Band Plot, and so we use that to render upper and lower in cells of Dashboard view.

Reply:
Keeping the index and using it to looking the query in props is the secret sauce

Reply:
imma need your help validating my port of this to notebooks then, cause they dont share that same schema. later though, and unrelated to this PR